ui/pages/confirmations/confirmation/templates/add-ethereum-chain.js

```diff
@@ -172,6 +189,9 @@ async function getAlerts(pendingApproval, data) {
     ) {
       alerts.push(MISMATCHED_NETWORK_RPC);
     }
+    if (DEPRECATED_NETWORKS.includes(pendingApproval.requestData.chainId)) {
+      alerts.push(DEPRECATED_CHAIN_ALERT);
+    }
```

Comment:
It will be nice to also add test coverage for this change.

Reply:
Thanks @jpuri  test added 🙏